modules/qenta/checkoutseamless/views/admin/de/qentacheckoutseamless_lang.php

```diff
@@ -0,0 +1,207 @@
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_SENDERBIC' => 'Sofort. Sender BIC',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_SENDERIBAN' => 'Sofort. Sender IBAN',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_SENDERCOUNTRY' => 'Sofort. Sender Land',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_SENDERCRITERIA' => 'Sofort. Sender Kriterien',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_MANDATEID' => 'SEPA Mandat-ID',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_MANDATESIGNATUREDATE' => 'SEPA Mandat Unterschriftsdatum',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_CREDITORID' => 'SEPA Kreditor-ID',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_DUEDATE' => 'SEPA F&auml;lligkeit',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_LANGUAGE' => 'Sprache',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_MESSAGE' => 'Nachricht',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_LIABILITYSHIFTINDICATOR' => 'liabilityShiftIndicator',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_INSTRUMENTCOUNTRY' => 'instrumentCountry',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_RISKINTERCEPT' => 'riskIntercept',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_RISKREASONCODE' => 'riskReasonCode',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_RISKREASONMESSAGE' => 'riskReasonMessage',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_RISKRETURNCODE' => 'riskReturnCode',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_RISKRETURNMESSAGE' => 'riskReturnMessage',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_CHECKOUTTYPE' => 'checkoutType',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_AVSRESPONSECODE' => 'avsResponseCode',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_AVSRESPONSEMESSAGE' => 'avsResponseMessage',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_AVSPROVIDERRESULTCODE' => 'avsProviderResultCode',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_AVSPROVIDERRESULTMESSAGE' => 'avsProviderResultMessage',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_IDEALCONSUMERNAME' => 'idealConsumerName',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_IDEALCONSUMERBIC' => 'idealConsumerBIC',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_IDEALCONSUMERCITY' => 'idealConsumerCity',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_IDEALCONSUMERIBAN' => 'idealConsumerIBAN',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_IDEALCONSUMERACCOUNTNUMBER' => 'idealConsumerAccountNumber',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_SENDERACCOUNTOWNER' => 'senderAccountOwner',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_SECURITYCRITERIA' => 'securityCriteria',
+    'ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_ANONYMOUSPAN' => 'anonymousPan',
+    'QCS_SUBMIT' => 'Senden',
+    'QCS_SUBMIT_CONFIG' => 'QENTA Checkout Seamless Konfiguration senden',
+    'MODULE_PAYMENT_QCS_EXPORT_CONFIG_RECEIVER' => 'Empf&auml;nger',
+    'MODULE_PAYMENT_QCS_EXPORT_CONFIG_CONFIG_STRING' => 'Konfiguration',
+    'MODULE_PAYMENT_QCS_EXPORT_CONFIG_DESC_TEXT' => 'Kommentar',
+    'MODULE_PAYMENT_QCS_EXPORT_CONFIG_RETURN_MAIL' => 'Antwort E-Mail-Adresse',
```

Comment:
ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_PROVIDERREFERENCENUMBER please add this key with translation in both languages: "Provider reference number" the same goes for ORDER_OVERVIEW_QENTA_CHECKOUT_SEAMLESS_PROVIDERCONTRACTNUMBER where the translation would be "Provider contract number"